accounts/templates/accounts/participant_list.html

```diff
@@ -44,16 +52,16 @@
    <div class="row mt-xl visible-md visible-lg visible-xl">
        <div class="col-md-3 col-md-offset-1">
            <strong> Username </strong>
-           <a class="pl-sm" aria-label="Sort ascending by username" href="{% url 'exp:participant-list' %}?{% query_transform request sort='given_name' %}"><i aria-hidden="true" class="fa fa-chevron-up" role="button" name="name"></i></a>
-           <a aria-label="Sort descending by username" href="{% url 'exp:participant-list' %}?{% query_transform request sort='-given_name' %}"><i aria-hidden="true" class="fa fa-chevron-down" role="button" name="name"></i></a>
+           <a class="pl-sm" aria-label="Sort ascending by username" href="{% url 'exp:participant-list' %}?{% query_transform request page='1' sort='given_name' %}"><i aria-hidden="true" class="fa fa-chevron-up" role="button" name="name"></i></a>
```

Comment:
Could you just change the `query_transform` tag to implicitly go to page one if sort is specified instead of passing it every time?